boards/riscv/it8xxx2_evb/doc/index.rst

```diff
@@ -0,0 +1,73 @@
+.. _it8xxx2_evb:
+
+ITE IT8XXX2 series
+######################
+
+Overview
+********
+
+The IT8XXX2 is a 32-bit RISC-V Micro-controller and a
+highly integrated embedded controller with system functions suitable for mobile system applications.
+To find out more about ITE, visit our World Wide Web at:
+`ITE's website <http://www.ite.com.tw>`_.
```

Comment:
Can you link to the actual product?

Reply:
We have prapared for it.
But it takes some time, we will solved other review first.